### DISCUSSION TITLE
should features and ideas be requested and proposed here, or do you prefer them in issues tab ?

### DISCUSSION THREAD

**Discussion by avichou** [sannidhyaroy/Soduto#35] [Ideas]
title pretty much. hope im not annoying with those questions, just trying to get the gist of git

**Comment by sannidhyaroy** [comment]
Both are pretty much fine, and people have different workflows of using git. Personally, I open a discussion if the issue or bug or pretty much any topic, needs to have a discussion with participants like decision making or sorting things out. If it's just a bug that I want to let the developer know, I'll just open an issue for it. If I can't decide, I'll again just make an issue and the owner of the repository may convert it to a discussion if they want to.
It's not really much to think about and moreover, you can convert issues to discussions and vice-versa.
